src/ls/Diagnostics.mli

```diff
@@ -0,0 +1,2 @@
+val diagnostics_of_results :
```

Comment:
Again, a small comment in the .mli will help.
The .mli are the most important files.
That's usually what we read when we want to use other people's code.